src/mudlet.h

```diff
@@ -70,7 +70,7 @@ class mudlet : public QMainWindow, public Ui::MainWindow
    static                        mudlet * self();
    // This method allows better debugging when mudlet::self() is called inappropriately.
    static                        void start();
-   HostManager *                 getHostManager();
+   HostManager& getHostManager() { return mHostManager; }
```

Comment:
Do we want/need to go the whole way and specify that the caller cannot modify the reference - or do we need to?

    const HostManager& getHostManager() const { return mHostManager; }

___If___ I have got this right this is the only time that marking a return value as `const` is valid - because, as a reference, it is "wired" to the original memory location and it is possible to ensure that it is not used to modify the referred to thing, a `HostManager`instance in this instance - and in that case a second `const` means that the reference used cannot be used to modify things in _this_ class.

OTOH Some of this C++ stuff sometimes goes over my head! :stuck_out_tongue_winking_eye: